src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -369,10 +351,8 @@ private static IEnumerable<MsalAccessTokenCacheItem> FilterByTokenType(Authentic
 
             if (filterByTenantId)
             {
-                tokenCacheItems = tokenCacheItems.FilterWithLogging(item =>
```

Comment:
I was under the impression that developers need to set logging to VERBOSE for the filters to enumerate the underlying collection? I think we should simply tell people that in perf heavy apps they should not enable verbose logs (pretty standard ask) and leave the logging alone.

Reply:
Done. Reverted back.